homeassistant/components/hue/sensor_base.py

```diff
@@ -31,8 +31,9 @@ def _device_id(aiohue_sensor):
 
 async def async_setup_entry(hass, config_entry, async_add_entities, binary=False):
     """Set up the Hue sensors from a config entry."""
+    sensor_key = CURRENT_SENSORS_FORMAT.format(config_entry.data["host"])
```

Comment:
Why not just use `config_entry.entry_id` ? We should key everything across config entries. I guess this works too .